app/src/main/java/cmput301f17t13/com/catisadog/activities/summary/AddHabitActivity.java

```diff
@@ -124,8 +124,9 @@ public void saveHabit(View v) {
         HashSet<Integer> schedule = new HashSet<>();
         for (int i = 0; i < days.size(); i++) {
             if (days.get(i).isSelected()) {
-                schedule.add(days.get(i).getCalendarDayId());
-                Log.d("asd", Integer.valueOf(i).toString());
+                // Hack to map the weekdaybuttons values to DateTimeConstants
+                if (i == 0) schedule.add(7);
```

Comment:
Not sure why you add 7 (Since that's Sunday).

Reply:
Sunday is 0 in the `weekdaybuttons` and 7 in `DateTimeConstants`. The two libraries aren't compatible with one another (this one took me a while to find).

Reply:
Ok, makes sense.